ui/segmented-bar.info/main.reel/main.js

```diff
@@ -0,0 +1,47 @@
+var Component = require("ui/component").Component;
+
+exports.Main = Component.specialize({
+    // SegmentedBar1
+    currentSegment: {value: null},
+    handleSegmentedBar1Action: {
+        value: function (event) {
+            console.log("heard");
+            this.currentSegment = event.detail.get('data').region;
+        }
+    },
+
+    // SegmentedBar2/3
+    constructor: {
+        value: function () {
+            this.content = [
+                {"region": "region1", "size": 10, "flag": true},
+                {"region": "region2", "size": 10, "flag": true},
+                {"region": "region3", "size": 10, "flag": true},
+                {"region": "region4", "size": 10, "flag": true},
+                {"region": "region5", "size": 10, "flag": true},
+                {"region": "region6", "size": 50, "flag": true}
+            ];
+            this.tree = {
+                region: "USA",
+                size: 99,
+                regions: [
+                    {
+                        region: "Canada",
+                        size: 20
+                    },
+                    {
+                        region: "Ivory Coast",
+                        size: 30,
+                        regions: [
+                            {
+                                region: "China",
+                                size: 40
+                            }
+                        ]
+                    }
+                ]
+            };
+
+        }
+    }
+});
```

Comment:
nit: Why is currentSegment all on one line, with no line separating it from the following function? It may save space but the inconsistency slows down reading.